src/resolvers/Mutation/acceptMembershipRequest.ts

```diff
@@ -65,7 +68,7 @@ export const acceptMembershipRequest: MutationResolvers["acceptMembershipRequest
     await adminCheck(context.userId, organization);
 
     const userIsOrganizationMember = organization.members.some(
-      (member) => member.toString() === user?._id.toString()
+      (member: string) => member.toString() === user?._id.toString()
```

Comment:
Wrong type annotation. Member wouldn't be a string, but a Mongoose Object ID (That's why you need to use the `.toString()` method)
Please fix the same.

Reply:
![image](https://user-images.githubusercontent.com/76558924/230713169-a2e723b8-5f3e-41f5-9726-a3b64c3c67d9.png)
it is giving error, if we doesn't define the type to it.